From 9d33599f12787f85a19e8147604f919ac40c02c1 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 4 Jun 2014 15:26:11 +0200
Subject: [PATCH] Revert "Use the "gcc" feature in simd.prf"

It's causing linker issues, see my comment in:
https://bugreports.qt-project.org/browse/QTBUG-38544

This reverts commit 3b00cc480400d47c59c29d3507c7fcdc9be2a489.
---
 mkspecs/features/simd.prf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/features/simd.prf b/mkspecs/features/simd.prf
index 62a63f9..4f86f60 100644
--- a/mkspecs/features/simd.prf
+++ b/mkspecs/features/simd.prf
@@ -18,7 +18,7 @@ QT_CPU_FEATURES = $$eval(QT_CPU_FEATURES.$$QT_ARCH)
 #
 # Set up compilers for SIMD (SSE/AVX, NEON etc)
 #
-gcc {       # includes intel_icc and clang variants
+*-g++*|intel_icc|*-clang*|*-qcc* {
     sse2 {
         HEADERS += $$SSE2_HEADERS
 
-- 
2.0.0

